src/core/lib/http/parser.cc

```diff
@@ -361,6 +361,13 @@ void grpc_http_request_destroy(grpc_http_request* request) {
   gpr_free(request->path);
 }
 
+void grpc_http_header_destroy(grpc_http_header* header) {
+  if (header != nullptr) {
+    gpr_free(header->key);
+    gpr_free(header->value);
+  }
+}
+
```

Comment:
Where is this method used? The headers are already cleaned in the request/response destroy.